analyzers/src/ITs.JsonParser/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CommandLine;
+using ITs.JsonParser;
+
+Parser.Default.ParseArguments<CommandLineOptions>(args).MapResult(Execute, errs => 1);
+
+int Execute(CommandLineOptions options)
```

Comment:
What's your plan with coverage? This probably should have a test project created with it. The coverage issue is now hidden by the failing UTs

Reply:
I didn't really think about it, since it was not covered beforehand either.
I am not sure how worth it it is to spend the time and add tests to this part of the codebase, what do you think?